src/Arcus.Templates.AzureFunctions.ServiceBus.Queue/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM mcr.microsoft.com/azure-functions/dotnet:4 AS base
+WORKDIR /app
+EXPOSE 80
+
+FROM mcr.microsoft.com/dotnet/sdk:6.0.100-bullseye-slim AS build
+WORKDIR /src
+COPY ["Arcus.Templates.AzureFunctions.ServiceBus.Queue.csproj", ""]
+
+COPY . .
+WORKDIR "/src/."
+RUN dotnet build "Arcus.Templates.AzureFunctions.ServiceBus.Queue.csproj" -c Release -o /app
+
+FROM build AS publish
+RUN dotnet publish "Arcus.Templates.AzureFunctions.ServiceBus.Queue.csproj" -c Release -o /app
+
+FROM base AS final
+WORKDIR /app
+COPY --from=publish /app .
+ENV AzureWebJobsScriptRoot=/app \
+    AzureFunctionsJobHost__Logging__Console__IsEnabled=true
```

Comment:
Just a question, is this a custom environment variable/setting ?  Or is this a documented variable for Azure Functions ?

Reply:
This is an built-in environment variable that was already part of the Dockerfile when it got generated. So, I guess we can leave it. 😄

Reply:
Sure. :) It just made me wonder, because I once ran into a situation where I had dan issue with environment-variables with double underscores, and then I heard from MSFT that it was not advised to use this in Azure Functions, but I have no idea about the underlying reason.